### PR TITLE
Small benchmark adjustments

### PR DESCRIPTION
**Pull request by danieljanes**

### CODE REVIEW

File: xain/benchmark/aggregation/final_task_accuracies.py [hunk 1]
```diff
@@ -112,9 +112,9 @@ def aggregate() -> str:
 
     fpath = plot(
         data,
-        title="Max achieved accuracy for unitary and federated learning",
-        xlabel="partitioning grade",
-        ylabel="accuracy",
+        title="",  # TODO
```
Comment (r-marques) [inline xain/benchmark/aggregation/final_task_accuracies.py]: Is this something we want to do in this pr?

File: xain/benchmark/aggregation/task_accuracies.py [hunk 2]
```diff
@@ -96,8 +96,8 @@ def aggregate() -> str:
 
     fpath = plot(
         data,
-        title="Validation set accuracy for unitary and federated learning",
-        ylabel="accuracy",
+        title="",  # TODO
```
Comment (r-marques) [inline xain/benchmark/aggregation/task_accuracies.py]: Same as above

File: xain/benchmark/aggregation/final_task_accuracies.py (hunk 1 above)
Comment (danieljanes) [reply xain/benchmark/aggregation/final_task_accuracies.py]: No, it's not for this PR. I left the comment in there to follow up on this - ideally we want to include a way in our benchmark "DSL" to configure this per group or per task

File: xain/benchmark/aggregation/task_accuracies.py (hunk 2 above)
Comment (danieljanes) [reply xain/benchmark/aggregation/task_accuracies.py]: Same as above

**Pull request merged**